collections/guides/quickstarts/quickstart_curl.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: Quickstart with cURL
+permalink: /getting_started/quickstart_with_curl/
+categories: quickstart
+nav_order: 2
+has_children: true
+has_toc: false
+description: Get Started with Basis Theory and cURL
+image:
+    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+    width: 1200
+    height: 630
+---
+# Quickstart with cURL
+
+In this tutorial, using cURL you will learn to use Basis Theory’s platform to secure a string of data and read the data back. Once you’ve completed this tutorial, you’ll have a foundational understanding of how to secure your data with Basis Theory allowing you to no longer worry about the underlying storage or encryption of the data.
+
+Looking for a Postman library?
+
+If you are already familiar with Basis Theory and our platform in general, our guide on [How to send Token data via HTTP](https://guides.basistheory.com/guides/use-token-data-in-http-requests/) or [Collecting payments with Elements](https://guides.basistheory.com/guides/collect-atomic-cards-with-elements/) may be a better place to start.
+
+## Step 1: Ensure cURL is ready on your system
+
+### Unix/Linux/macOS:
+
+Your system most likely has cURL already installed and you can check with the following command:
+
+```shell
+curl --help
+```
+
+### Windows
+There is a chance that if you have Windows 10 installed, you already have cURL. If the following command doesn’t work — here is help to get [cURL installed](https://stackoverflow.com/a/16216825).
+
+```shell
+curl --help
+```
+
+## Step 2: Create a new Application in your Basis Theory Account
+
+If you don’t already have a Basis Theory account and your first Tenant, create one [here](https://portal.basistheory.com/register).
+
+Within your Basis Theory account, create a new Application. To create a new application, head to our portal [here](https://portal.basistheory.com/applications/create) — this Application’s API Key will enable you to authenticate with the Basis Theory platform and create Tokens within your Tenant.
+
+### 2.1 Enter an Application Name
+This name allows you to identify this application in the future — for this tutorial enter “Ruby Quickstart”.
+
+<img src="/assets/images/getting_started/application_name_curl.png" />
+
+### 2.2 Select Server-to-Server Application Type
+The Server-to-Server Application Type enables server-side applications to integrate with the Basis theory Platforms directly.
+
+<img src="/assets/images/getting_started/application_type.png" />
+
+### 2.3 Select Permissions
+Select `token:general:create` and `token:general:read` [permissions](https://docs.basistheory.com/api-reference/#permissions-permission-object) with the default “Low Impact”. These two permissions allow your Application to create a new Token and read the value back when you need to access it.
```

Comment:
```suggestion
Select `token:general:create` and `token:general:read` [permissions](https://docs.basistheory.com/api-reference/#permissions-permission-object) with “High Impact”. These two permissions allow your Application to create a new Token and read the value back when you need to access it.
```

We will need `token:general:read:high` permission to be able to read back the plaintext token data in this guide.